### PR TITLE
Update documentation to 2022b+

### PR DESCRIPTION
**Pull request by galou**
Also fix a typo.

Signed-off-by: Gaël Écorchard <gael.ecorchard@cvut.cz>

**Description**
Fix the PROTO example to be compatible with R202b and more recent.

**Documentation**
https://cyberbotics.com/doc/guide/getting-started-with-webots?version=galou:update_doc

**Screenshots**
The view point is slightly different. I attach a new screenshot.
three_chairs_proto_example

### CODE REVIEW

**Comment by galou** [comment]
Side comment.

I did the PR the way I know. As a committer, I guess I can do it another way but I don't know which.

File: docs/reference/proto-example.md
Comment (omichel) [inline docs/reference/proto-example.md:100]:
```suggestion
#VRML_SIM {{ webots.version.major }} utf8

EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/{{ webots.version.major }}/projects/appearances/protos/DamascusSteel.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/{{ webots.version.major }}/projects/appearances/protos/Parquetry.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/{{ webots.version.major }}/projects/appearances/protos/VarnishedPine.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/{{ webots.version.major }}/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto"
EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/{{ webots.version.major }}/projects/objects/floors/protos/Floor.proto"
EXTERNPROTO "../protos/TwoColorChair.proto"

```

**Review by omichel — CHANGES_REQUESTED** [review]
Thank you.
It is slightly better to use the version macro, see my change request.

**Comment by omichel** [comment]
Also, for the documentation it is better to target the `released` branch instead of the `master` branch, so that it gets fixed online right after the PR is merged.

**Comment by omichel** [comment]
> I did the PR the way I know. As a committer, I guess I can do it another way but I don't know which.

Yes, you could have created a branch on the webots repo. Now you have the rights for this.
Simply when you modify a file online directly on the master or the released branch for example, GitHub will propose you to create your own local branch on the webots repo.

**Comment by galou** [comment]
Closing in favor of #5782.

**Pull request closed without merging**
